src/mlpack/tests/random_forest_test.cpp

```diff
@@ -558,3 +558,54 @@ TEST_CASE("WarmStartTreesPredictionsQualityTest", "[RandomForestTest]")
 
   REQUIRE(newCorrect - oldCorrect >= 0);
 }
+
+/**
+ * Ensure that the Extra Trees algorithm gives decent accuracy.
+ */
+TEST_CASE("ExtraTreesAccuracyTest", "[RandomForestTest]")
```

Comment:
Did you test the robustness of this test? I ran it about 50 times and saw a handful of failures with accuracies as low as about 92%. I think the algorithm is working fine, but maybe the test tolerances should be adjusted. We should aim for no failures in 1000 runs. (To test that be sure to use different random seeds each time. :+1:)

Reply:
Hey, @rcurtin I tested for 600 runs after reducing accuracy and it didn't fail. In my last experiment, I forgot to change the random seed and I was getting the same accuracy in each run which I couldn't see because I was grepping the output only when it failed.

Reply:
No worries, sounds good! Thanks for the fix. :)